image_proc/include/visibility.h

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2008, Willow Garage, Inc.
```

Comment:
At least this used to be the case. Good to make it consistent either way.

Reply:
Let's see if the Intel people (I think they are) want to put their copyright on it since the rewrote most of it, but I'll change for now. I agree with the point on consistency.